A3ServerTool/Helpers/UniversalParser.cs

```diff
@@ -364,13 +365,16 @@ public string ParseArrayProperty(int[] valueAsArray)
         private Dictionary<string, string> ConvertFromTextToDictionary(string[] textProperties, Type type)
         {
             var nameToValueDictionary = new Dictionary<string, string>();
+            var parent = "";
 
             for (int i = 0; i < textProperties.Length; i++)
             {
                 var splittedProperty = textProperties[i].Split('=')
                     .Where(x => x != "=")
                     .Select(x => x.Trim())
                     .ToArray();
+                if (splittedProperty[0].StartsWith("class ")) parent = splittedProperty[0].Substring("class ".Length);
+                if (splittedProperty[0].StartsWith("};")) parent = "";
```

Comment:
Should multiple parent classes be handled here?

Reply:
Yes